tests/test_processing.py

```diff
@@ -156,27 +156,27 @@ def test_article_html_generator(self):
     @patch("documentstore_migracao.processing.generation.article_html_generator")
     def test_article_ALL_html_generator(self, mk_article_html_generator):
 
-        with utils.environ(CONVERSION_PATH=SAMPLES_PATH, VALID_XML_PATH=""):
+        with utils.environ(CONSTRUCTOR_PATH=SAMPLES_PATH):
             generation.article_ALL_html_generator()
             mk_article_html_generator.assert_called_with(ANY)
             self.assertEqual(
                 len(mk_article_html_generator.mock_calls), COUNT_SAMPLES_FILES
             )
 
-    @patch("documentstore_migracao.processing.generation.article_html_generator")
-    def test_article_ALL_html_generator_with_exception(self, mk_article_html_generator):
+    # @patch("documentstore_migracao.processing.generation.article_html_generator")
```

Comment:
@cesarbruschetta este test não é necessário?

Reply:
@joffilyfe o que acontece  com os teste:
* `test_reading_article_ALLxml_with_exception`
* `test_article_ALL_html_generator_with_exception`

É que o comportamento do metodo alvo (`reading.reading_article_ALLxml`e `generation.article_ALL_html_generator`) foi alterado e não esta mais retornando o `raise Error` que era esperado, achei melhor remover esses teste pois acredito que na verdade o teste não testa nada, sinceramente, precisamos rever como testar de verdade esse ponto, pois na verdade existia mais esses teste para aumentar a quantidade de linhas cobertas pelo coverage, então por isso existe muitos _mock_ para forçar que o certas linhas do código sejam percorridas pelos testes.

Então por agora achei melhor remover o teste para nao termos problema na integração com o https://travis-ci.org

Reply:
não seria melhor remover o teste em vez de apenas comentar?